### DISCUSSION TITLE
Is there any reason why in V3 "Back" has now become "Lats" as a default?

### DISCUSSION THREAD

**Discussion by Testah2024** [WhyAsh5114/MyFit#64] [General]
Most Back exercises (Such as Pull Downs, Pull Ups, Rows, etc) target Lats, Rhomboid, Teres, Erector, etc all at once.

**Comment by WhyAsh5114** [comment]
The reason was to separate close grip pulling (primarily Lats, with teres, etc.) from wide grip pulling (primarily traps, with rear delts, etc.)

I saw that many people (even I) mostly divide the back exercises into "width" (lats) and "thickness" (traps), which is why I thought this would be better suited.

Maybe this isn't the best approach, do you have any suggestions?
